### PR TITLE
fix: react ad display

### PR DESCRIPTION
**Pull request by parkjh**
- adding checks to disable ads if neccessary
- setting server side prebid config (copied from testad-config.json from render)
- add settimeout on setting targets on prebid slots
- removed pixel ad components from article skeleton component
- accepts global ad slots from render
- ad-init defines slots for all the ads on the page

### CODE REVIEW

**Comment by tuncaulubilge** [comment]
Can you add some description, and ideally a jira ticket link?

**Comment by parkjh** [comment]
@tuncaulubilge will do

**Comment by aronshamash** [comment]
need to use https://www.conventionalcommits.org/en/v1.0.0-beta.2/ in your commit msgs

File: packages/ad/__tests__/utils/ad-init.js [hunk 1]
```diff
@@ -30,7 +31,7 @@ export default () => {
     expect(init2.initPageAsync).toHaveBeenCalledTimes(0);
   });
 
-  it("performs slot-level setup for every slot", () => {
+  xit("performs slot-level setup for every slot", () => {
```
Comment (tuncaulubilge) [inline packages/ad/__tests__/utils/ad-init.js]: Typo on xit?

Comment (aronshamash) [reply packages/ad/__tests__/utils/ad-init.js]: looks commented out

File: packages/ssr/src/standalone-renderer/app.js [hunk 2]
```diff
@@ -9,9 +8,16 @@ const logger = require("../lib/simple-logger");
 
 const port = 3000;
 const server = express();
+const graphqlApiUrl = process.env.GRAPHQL_ENDPOINT;
```
Comment (colincclark) [inline packages/ssr/src/standalone-renderer/app.js]: most of this refactor seems to belong in a separate PR

**Comment by colincclark** [comment]
can you ensure PRs are more atomic. The yarn commands, ssr changes and extra yarn scripts are all separate PRs

File: packages/ad/fixtures/article-ad-config.json [hunk 3]
```diff
@@ -50,5 +50,6 @@
       }
     }
   },
-  "bidderSlots": ["ad-header", "ad-article-inline"]
+  "bidderSlots": ["header", "ad-article-inline"],
```
Comment (colincclark) [inline packages/ad/fixtures/article-ad-config.json]: why header and not ad-header? all the other slots specify ad in the name

File: packages/ad/__tests__/utils/ad-init.js (hunk 1 above)
Comment (colincclark) [reply packages/ad/__tests__/utils/ad-init.js]: no commented out tests please

Comment (parkjh) [reply packages/ad/__tests__/utils/ad-init.js]: @tuncaulubilge @aronshamash it was removed as it was no longer the functional behaviour

File: packages/ad/src/utils/generate-config.js [hunk 4]
```diff
@@ -20,8 +20,11 @@ const getSizeMaps = slotName => {
     case "ad-header":
```
Comment (colincclark) [inline packages/ad/src/utils/generate-config.js]: why do we have header and ad-header - what is the difference?

File: packages/article-magazine-comment/__tests__/web/__snapshots__/article-with-style.web.test.js.snap [hunk 5]
```diff
@@ -744,9 +744,6 @@ exports[`full article with style 1`] = `
         <RelatedArticles />
       </aside>
     </div>
-    <Ad />
```
Comment (colincclark) [inline packages/article-magazine-comment/__tests__/web/__snapshots__/article-with-style.web.test.js.snap]: are these supposed to be removed?

File: packages/ssr/src/standalone-renderer/app.js (hunk 2 above)
Comment (parkjh) [reply packages/ssr/src/standalone-renderer/app.js]: @colincclark all the ssr standalone server changes are now reverted, but the rest are just config changes for ads that are required

File: packages/ssr/package.json [hunk 6]
```diff
@@ -39,7 +39,7 @@
     "@times-components/eslint-config-thetimes": "0.8.8",
     "@times-components/fixture-generator": "0.0.3",
     "@times-components/mock-tpa-server": "0.1.17",
-    "cypress": "3.1.0",
+    "cypress": "3.1.3",
```
Comment (colincclark) [inline packages/ssr/package.json]: why? separate PR?

**Comment by parkjh** [comment]
@aronshamash split out the prs
https://github.com/newsuk/times-components/pull/1559
https://github.com/newsuk/times-components/pull/1560

File: packages/ad/fixtures/article-ad-config.json (hunk 3 above)
Comment (parkjh) [reply packages/ad/fixtures/article-ad-config.json]: @colincclark nope this is not the case, as the html id of the ad slots were different originally

File: packages/article-magazine-comment/__tests__/web/__snapshots__/article-with-style.web.test.js.snap (hunk 5 above)
Comment (parkjh) [reply packages/article-magazine-comment/__tests__/web/__snapshots__/article-with-style.web.test.js.snap]: @colincclark yes, as part of this pr I've remove the global pixel ads from article skeleton components

File: packages/ssr/package.json (hunk 6 above)
Comment (parkjh) [reply packages/ssr/package.json]: @colincclark see @aronshamash request

File: packages/ad/__tests__/utils/ad-init.js (hunk 1 above)
Comment (tuncaulubilge) [reply packages/ad/__tests__/utils/ad-init.js]: We can just delete this then :+1:

**Comment by parkjh** [comment]
@colincclark @aronshamash would it be possible for this pr to be reviewed today?

**Comment by parkjh** [comment]
https://app.bitrise.io/build/58f3de953ac7074b not being picked up...
